index.html

```diff
@@ -97,32 +117,226 @@
 					        	<a class="nav-link" href="#">נקודות שירות
 					        	</a>
 					        </li>
-					        <li class="nav-item active">
-					        	<a class="nav-link" href="#">אודות
-					        	</a>
-					        </li>
 			    		</ul>
-	    			</div>
-	    			<div class="paz-logo-nav-2">
-					        	<img src="img/paz-logo-nav-2.png">
+			    	</div>
+		    		<div class="paz-logo-nav-2">
+				        <img src="img/pazomat-logo.svg"
+    					 class="pazomat-logo">
 					</div>
-	    		</div>
-	    	</div>
-    </header>
-   	
-    	
-
-
-
-
-
-
-
-
-    	<!-- TOP SECTION -->
-    	<section>
+    			</div>
+			</div>
+		</div>
+	</header>
+	
+	<!-- TOP SECTION -->
+	<section class="container top-section">
+		<div class="gallery">
+			<div class="Locating-stations">
+				<div class="swiper-container">
+				    <!-- Additional required wrapper -->
+				    <div class="swiper-wrapper">
+				        <!-- Slides -->
+				        <div class="swiper-slide"><img src="img/main-gallery-slide-1.png"></div>
```

Comment:
Nope. That's not the way to do it... The whole thing should swipe, the slide content as well
Heres an image of the slide content:
![image](https://user-images.githubusercontent.com/3458336/33445688-4dbdf750-d606-11e7-968c-4a54a451b8df.png)
How to fix:

- The slide content (`gallery-p`) should sit inside the `swiper-slide` element.
- The image should not be an `img` element but a `background-image` property on the `swiper-slide` element. This way you can just position the content inside the slide without worrying about the image colliding with the content